src/qibolab/instruments/zhinst.py

```diff
@@ -499,22 +480,61 @@ def Drive(self, exp, qubits):
                             exp.play(signal=f"drive{qubit.name}", pulse=pulse.zhpulse, phase=pulse.pulse.relative_phase)
                             i += 1
 
+    # def Measure(self, exp, qubits):
+    #     # qubit readout pulse and data acquisition
+    #     play_after = None
+    #     if "drive" in str(self.sequence):
+    #         play_after = f"sequence_drive"
+    #     with exp.section(uid=f"sequence_measure", play_after=play_after):
+    #         for qubit in qubits.values():
+    #             if "c" in str(qubit.name):
+    #                 pass
+    #             else:
+    #                 if self.sequence[f"drive{qubit.name}"]:
+    #                     last_drive_pulse = self.sequence[f"drive{qubit.name}"][-1]
+    #                     time = round(last_drive_pulse.pulse.finish * 1e-9, 9)
+    #                 else:
+    #                     time = 0
+    #                 i = 0
+    #                 for pulse in self.sequence[f"readout{qubit.name}"]:
+    #                     exp.delay(signal=f"measure{qubit.name}", time= round(pulse.pulse.start * 1e-9, 9) - time)
+    #                     time += round(pulse.pulse.duration * 1e-9, 9) + round(pulse.pulse.start * 1e-9, 9) - time
+    #                     pulse.zhpulse.uid = pulse.zhpulse.uid + str(i)
+    #                     if isinstance(pulse, ZhSweeper):
+    #                         self.play_sweep(exp, qubit, pulse, section="measure")
+    #                     else:
+    #                         exp.play(
+    #                             signal=f"measure{qubit.name}", pulse=pulse.zhpulse, phase=pulse.pulse.relative_phase
+    #                         )
+    #                     # FIXME: This should be defined by the user as a pulse elsewhere
+    #                     # Is this a qibo pulse ???
+    #                     weight = lo.pulse_library.const(
+    #                         uid="weight" + pulse.zhpulse.uid,
+    #                         length=round(pulse.pulse.duration* 1e-9, 9),
+    #                         amplitude=1.0,
+    #                     )
+    #                     exp.acquire(signal=f"acquire{qubit.name}", handle=f"sequence{qubit.name}", kernel=weight)
+    #                     i += 1
```

Comment:
If possible, avoid large bunches of commented code: either you need and keep, or drop it (of course it might be useful for debugging, but consider that it makes more difficult to understand what's going on for everyone reading the diffs).

Reply:
I will keep it on a separete file for now, it's for something I need to test but I can't right now